### PR TITLE
attribution flag now resizes with font-size changes

### PR DESCRIPTION
**Pull request by sumitsaurabh927**
Tested on Chrome on Fedora and things are working fine. I changed font-size in Chrome's console and the attribution flag changed accordingly.

### CODE REVIEW

File: src/control/Control.Attribution.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as DomEvent from '../dom/DomEvent';
 import * as DomUtil from '../dom/DomUtil';
 import Browser from '../core/Browser';
 
-var ukrainianFlag = '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="12" height="8"><path fill="#4C7BE1" d="M0 0h12v4H0z"/><path fill="#FFD500" d="M0 4h12v3H0z"/><path fill="#E0BC00" d="M0 7h12v1H0z"/></svg>';
+var ukrainianFlag = '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="12" height="8" + viewBox="0 0 12 8" class="leaflet-attribution-flag"><path fill="#4C7BE1" d="M0 0h12v4H0z"/><path fill="#FFD500" d="M0 4h12v3H0z"/><path fill="#E0BC00" d="M0 7h12v1H0z"/></svg>';
```
Comment (Malvoz) [inline src/control/Control.Attribution.js:9]: Copy paste error? `+` Should not be included here.

**Review by Malvoz — CHANGES_REQUESTED** [review]
IE treats `height: auto` on the SVG differently from modern browsers such as Chrome and Firefox, resulting in a visual bug. Will need to resolve this.

**Comment by sumitsaurabh927** [comment]
I've removed `+` sign in the latest commit. Unfortunately I don't have access to IE. How else can I contribute? When will my PR be merged?

**Review by mourner — CHANGES_REQUESTED** [review]
Please only add the necessary changes to fix the bug, don't make unrelated changes — the `vertical-align` was there for a reason but was removed, and `!important` is necessary too. Also please keep formatting consistent.

**Comment by sumitsaurabh927** [comment]
I'm so sorry. I've fixed my mistakes and added back `vertical-align` and `!important`. I'm presuming the comment was the issue with consistency and have removed it. Shall I push these changes?

**Comment by Falke-Design** [comment]
Yes always push your changes if you are sure that they are ready to review

**Comment by sumitsaurabh927** [comment]
Done. Thanks for the tip!

File: dist/leaflet.css [hunk 2]
```diff
@@ -423,10 +423,12 @@ svg.leaflet-image-layer.leaflet-interactive path {
 .leaflet-control-attribution a:focus {
 	text-decoration: underline;
 	}
-.leaflet-control-attribution svg {
+.leaflet-attribution-flag{
```
Comment (mourner) [inline dist/leaflet.css:426]:
```suggestion
.leaflet-attribution-flag {
```

**Review by mourner — COMMENTED** [review]
The changes looks good now — let's see if we can find a way to resolve the issue in IE, maybe @Malvoz has ideas?

File: dist/leaflet.css [hunk 3]
```diff
@@ -423,10 +423,12 @@ svg.leaflet-image-layer.leaflet-interactive path {
 .leaflet-control-attribution a:focus {
 	text-decoration: underline;
 	}
-.leaflet-control-attribution svg {
+.leaflet-attribution-flag {
 	display: inline !important;
 	vertical-align: baseline !important;
-	}
+	width: 1em;
+	height: auto;
```
[the remark this answers is not in the document]
Comment (sumitsaurabh927) [reply dist/leaflet.css]: Done and pushed!

[the remark this answers is not in the document]
Comment (Malvoz) [reply dist/leaflet.css]: The height value in https://github.com/Leaflet/Leaflet/pull/8183/commits/c554d64f8e5ca4414983f68f5872cbbe350ff150 computes to `8.025px` (on normal document font size, on my device). Don't know if this is bothering anyone. The suggested value here would compute to `8px` in the same scenario.

@mourner?

[the remark this answers is not in the document]
Comment (Malvoz) [reply dist/leaflet.css]: @sumitsaurabh927 Did you mean to change the value from the [suggested](https://github.com/Leaflet/Leaflet/pull/8183#discussion_r857570861) one?

[the remark this answers is not in the document]
Comment (sumitsaurabh927) [reply dist/leaflet.css]: I originally set it to auto but you [suggested](https://github.com/Leaflet/Leaflet/pull/8183#discussion_r857570861) it to be `0.669em` so I set it to that. ( I thought it fixed height bug in IE)

[the remark this answers is not in the document]
Comment (Malvoz) [reply dist/leaflet.css]: I suggested `0.6669em`, but you set it to `0.669em`.

**Comment by sumitsaurabh927** [comment]
I'm so sorry for this! Stupid mistake 🤦‍♂️
Rectified it and pushed!

File: dist/leaflet.css [hunk 4]
```diff
@@ -423,10 +423,12 @@ svg.leaflet-image-layer.leaflet-interactive path {
...
 	text-decoration: underline;
 	}
-.leaflet-control-attribution svg {
+.leaflet-attribution-flag {
 	display: inline !important;
 	vertical-align: baseline !important;
-	}
+	width: 1em;
+	height: 0.6669em;
+}
```
[the remark this answers is not in the document]
Comment (sumitsaurabh927) [reply dist/leaflet.css]: Sorry this escaped my attention, fixed and pushed!

**Review by mourner — APPROVED** [review]

**Comment by mourner** [comment]
Thank you!

**Comment by sumitsaurabh927** [comment]
Thank you! This was my first contribution to an open source and needless to say I'm beyond happy that it has been merged. I'm so sorry for all the silly mistakes I did and promise that I'll keep them in mind whenever I'm contributing to any OS project.
Thanks you! ❤️

**Comment by mourner** [comment]
@sumitsaurabh927 no worries at all, you don't have to apologize so much — even contributors with decades of experience can easily make mistakes in PRs, and yours were very minor. You did well!

**Comment by sumitsaurabh927** [comment]
Okay. Thank you! 😊

**Pull request merged**